packages/fuel-indexer-macros/src/schema.rs

```diff
@@ -24,8 +24,23 @@ type ProcessedFieldResult = (
     proc_macro2::Ident,
     proc_macro2::Ident,
     proc_macro2::TokenStream,
+    Option<bool>,
```

Comment:
I thought about it but I didn't want to start throwing a whole bunch of enums around; upon revisiting, it would definitely be more self-documenting if we used an enum here instead. I'll make the adjustment in the next PR.